crates/astria-composer/src/ds/wire_format.rs

```diff
@@ -0,0 +1,11 @@
+use ethers::types::Transaction as EthersTx;
```

Comment:
Why aliases?

Reply:
The reasoning behind this is that every blockchain will have a Transaction type, but we will eventually support other Eth libraries. Plus typing `etheres::types::Transaction` everywhere to disambiguate is annoying. But I can remove it if you feel strongly